### PR TITLE
[MM-30788] Add FAQ section

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

For the joy of @agnivade :grin:  
PR adds FAQ and Troubleshoot sections. From now on it will be easier to keep track of common issues/questions.

Please feel free to remind me of more things to add because I am surely missing a lot.
Also let me know if you'd prefer a link to these in the README.

#### Ticket

https://mattermost.atlassian.net/browse/MM-30788

### CODE REVIEW

**Comment by agnivade** [comment]
@agnivade is overjoyed :) Yes, I'd prefer a link to this in the README.

**Review by isacikgoz — APPROVED** [review]
Nice one 👍

File: README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ There are mainly two ways to run a load-test:
 - On a Terraform deployment. This is the recommended way to start a load-test for production. [Link to the guide](docs/terraform_loadtest.md)
 - Locally. This is a good way to getting started and better understand the inner mechanics. [Link to the guide](docs/local_loadtest.md)
 
+## FAQ
+
+Answers to frequent questions and troubleshoot for common issues can be found in our [FAQ](docs/faq.md) section.
```
Comment (agnivade) [inline README.md:22]: troubleshooting?

File: docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## Troubleshoot
+
+### Users are not connecting
+
+Make sure that both `Enable Account Creation` and `Enable Open Server` settings are set to `true` in MM System Console.
+
+### Agent logs show several `current team should be set` errors.
+
+This can be caused by the app server not being initialized (at least one open team should be created). This can be done manually or through the `ltctl loadtest init` command.
+Also the `Max Users Per Team` setting in Mattermost System Console should be enough to account for the number of simulated users.
```
Comment (agnivade) [inline docs/faq.md:28]: Do we have the error handy for this one? Might help people googling for errors.

File: docs/faq.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+The rule of thumb is that when starting an unbounded load-test we should always shoot for more users than what we think an installation can support.
+`ClusterConfig.MaxActiveUsers`  should be set to  `AgentInstanceCount * UsersConfiguration.MaxActiveUsers`.
+
+## Troubleshoot
+
+### Users are not connecting
+
+Make sure that both `Enable Account Creation` and `Enable Open Server` settings are set to `true` in MM System Console.
+
+### Agent logs show several `current team should be set` errors.
```
Comment (agnivade) [inline docs/faq.md:25]: Wasn't there a similar error like `no team found` in the memstore? Should we add that too?

**Review by agnivade — COMMENTED** [review]
Looks great. Just some small comments.

File: docs/faq.md (hunk 2 above)
Comment (streamer45) [reply docs/faq.md:28]: If the max users per team is not set properly the error is the same. Users will first fail with `no team to join` and then `current team should be set`.

File: docs/faq.md (hunk 3 above)
Comment (streamer45) [reply docs/faq.md:25]: I think there was a bug at some point which we fixed. Now the error would be `current team should be set` if we expect the team to be there but none found.

**Review by agnivade — APPROVED** [review]

**Pull request merged**